contracts/WhitelistedRecipientsRegistry.sol

```diff
@@ -0,0 +1,142 @@
+// SPDX-FileCopyrightText: 2021 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.8.4;
+
+import "OpenZeppelin/openzeppelin-contracts@4.3.2/contracts/access/AccessControl.sol";
+import "./LimitsChecker.sol";
+
+/// @author psirex, zuzueeka
+/// @title Registry of allowed addresses for payouts
+/// @notice Stores list of whitelisted addresses
+contract WhitelistedRecipientsRegistry is AccessControl, LimitsChecker {
+    // -------------
+    // EVENTS
+    // -------------
+    event WhitelistedRecipientAdded(address indexed _whitelistedRecipient, string _title);
+    event WhitelistedRecipientRemoved(address indexed _whitelistedRecipient);
+
+    // -------------
+    // ROLES
+    // -------------
+    bytes32 public constant ADD_WHITELISTED_RECEPIENT_ROLE =
```

Comment:
a typo in `RECEPIENT` throughout the contract

Reply:
solved